plugins/backstage-plugin-flux/package.json

```diff
@@ -4,6 +4,12 @@
   "main": "src/index.ts",
   "types": "src/index.ts",
   "license": "MPL-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/weaveworks/weaveworks-backstage.git",
+    "directory": "plugins/backstage-plugin-flux"
+  },
+  "homepage": "https://docs.gitops.weave.works/docs/backstage/intro/",
```

Comment:
Not sure what this should be, we could also omit if this doesn't feel quite right.

Reply:
Given how often we move our docs around, maybe we leave it out?

Reply:
Or link to the GitHub Repo?